.github/workflows/reusable_run_notebook.yml

```diff
@@ -26,7 +26,7 @@ permissions:
 jobs:
   run-notebook:
     name: Run notebook
-    runs-on: ubuntu-latest
+    runs-on: buildjet-8vcpu-ubuntu-2204 # This image runs occasionally out of memory, BuildJet runners have more disk available.
```

Comment:
We could probably use the 16-core runner here instead. Those also have much more disk space available:

https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#machine-sizes-for-larger-runners

![image](https://github.com/user-attachments/assets/7d0de9fe-183b-4838-b84c-56669c4633e5)